scripts/performance/summarize

```diff
@@ -129,8 +232,13 @@ class Summarizer(object):
 def main():
     parser = argparse.ArgumentParser(usage=__doc__)
     parser.add_argument(
-        '-f', '--benchmark-file', required=True,
-        help='The CSV output file from the benchmark script.')
+        '-f', '--benchmark-files', required=True, nargs='+',
```

Comment:
Given benchmark files are the main thing you need for this script to work, it seems more idiomatic CLI to just have these be args rather than options.